app/pdfs/timelogs_pdf.rb

```diff
@@ -0,0 +1,31 @@
+require 'prawn'
+require 'prawn/table'
+class TimelogsPdf < Prawn::Document
+  COLUMNS = ["ID", "Start time", "Duration", "End time", "Trello card"]
+
+  def initialize(timelogs)
+    super()
+    @timelogs = timelogs
+    unless @timelogs.empty?
+      header
+      fill_document
+    else
+      text("This user has no timelogs yet")
+    end
+  end
+
+  private
+    def header
+      text("Timelogs\nOwner: #{@timelogs.first.user.email}")
+    end
+
+    def fill_document
+      content = [[]]
```

Comment:
I think it should be `content = []`
Also, you can do like this `content = [COLUMNS]`, which is much cleaner.